src/components/addSourceWizard/descriptions/HybridCommittedSpendDescription.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import { Stack } from '@patternfly/react-core';
+import { useIntl } from 'react-intl';
+
+import Point from './Point';
+import { ACCOUNT_AUTHORIZATION } from '../../constants';
+
+const HybridCommittedSpendDescription = ({ id }) => {
+  const intl = useIntl();
+  const { getState } = useFormApi();
+
+  const values = getState().values;
+
+  const isEnabled =
```

Comment:
Can you wrap this variable init in a `useMemo` hook? We don't want to run the same condition on each render. I don't expect the ID to change very often.